pkg/tuned/change-profile.jsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "react",
+], function(React) {
+
+"use strict";
+
+/* Performance profile entry
+ * Expected props:
+ *  - name (key)
+ *  - recommended (boolean)
+ *  - active (boolean)
+ *  - selected (boolean)
+ *  - title
+ *  - description
+ *  - click (callback function)
+ */
+var TunedDialogProfile = React.createClass({
+    render: function() {
+        var classes = "list-group-item";
+        if (this.props.selected)
+            classes += " active";
+        var recommended;
+        if (this.props.recommended)
+            recommended = <span className="badge pull-right">recommended</span>;
```

Comment:
using `_("recommended")`, since the `translatable` tag doesn't work correctly when used in js.